### PR TITLE
refactor: remove laravel passport

### PR DESCRIPTION
**Pull request by asbiin**
You fool, don't forget these steps:

- [x] Unit tests
- [ ] Tests with Cypress
- [ ] Documentation
- [ ] Dummy data

### CODE REVIEW

**Comment by djaiss** [comment]
Bravo!

File: .gitignore [hunk 1]
```diff
@@ -4,18 +4,16 @@
...
+.env.cypress.local
 .phpunit.result.cache
+.php_cs.cache
 Homestead.json
-.idea
 Homestead.yaml
 npm-debug.log
 yarn-error.log
-.php_cs.cache
-.DS_Store
```
Comment (djaiss) [inline .gitignore:17]: @asbiin Why did you remove that? 😢 

This files appears all the time on macos.

Comment (asbiin) [reply .gitignore:17]: It's a platform specific ignore, and should not be there.
